bin/cli.sh

```diff
@@ -0,0 +1,177 @@
+    fi
+fi
+
+function usage()
+{
+    echo "Usage: cmd [-c] [-b <admin|developer|user>] [-v <go|xml|json|php|kotlin|yaml>] -r <revision> -d <release date>"
+}
+
+function clean_build_dir()
+{
+    echo "Cleaning build directory..."
+    rm -rvf "build/"
+    echo "...build directory cleaned."
+}
+
+function validate_code_files()
+{
+    language="$1"
+
+    case $language in
+        go)
+            command -v golint &>/dev/null && \
+                echo "Validating go source files" && \
+                find . -type f -name "*.go" \
+                    ! -path "./node_modules/*" \
+                    ! -path "**/vendor/*" \
+                    ! -path "./.git/*" \
+                    -exec sh -c 'echo Linting {} && golint {} && echo' \;
+            ;;
+
+        json)
+            command -v jsonlint &>/dev/null && \
+                find . -type f -name "*.json" \
+                    ! -path "./node_modules/*" \
+                    ! -path "**/vendor/*" \
+                    ! -path "./.git/*" \
+                    -exec sh -c 'echo Linting {} && jsonlint -qp {} && echo' \;
+            ;;
+
+        kotlin)
+            command -v ktlint &>/dev/null && \
+                ktlint --reporter=plain "./modules/*_manual/**/*.kt" || true;
+            ;;
+
+        php)
+            command -v php &>/dev/null && \
+                find ./modules/*_manual/examples -type f -name "*.php" -exec php -l {} \;
+            ;;
+
+        xml)
+            command -v xmllint &>/dev/null && \
+                find ./modules/*_manual/examples -type f -name "*.xml" -exec xmllint --noout {} \;
+            ;;
+
+        yaml)
+            command -v yamllint &>/dev/null && \
+                find . -type f -name "*.yml" \
+                    ! -path "./node_modules/*" \
+                    ! -path "**/vendor/*" \
+                    ! -path "./.git/*" \
+                    -exec sh -c 'echo Linting {} && yamllint -f parsable {} && echo' \;
+            ;;
+
+        *)
+            echo "That language is not, currently, supported" 
+            exit $ERR_UNSUPPORTED_LANGUAGE
+            ;;
+    esac
+}
+
+function convert_antora_nav_to_asciidoc_list()
+{
+    filename="$1"
+
+    while read line 
+    do
+        if [[ ${line} =~ \]$ ]]; then 
+            level_offset=$(echo "$line" | awk -F"*" '{print NF-1}')
+            revised_line=$(echo "$line" | sed 's/xref:/include::{module_base_path}/' | sed 's/\[.*\]//g' | sed -r 's/^\*{1,} //')
+            echo "${revised_line}[leveloffset=+${level_offset}]"
+        fi
+    done < "${filename}"
+}
+
+function build_pdf_manual()
+{
+    manual="$1"
+    release_date="$2"
+    revision="$3"
+    build_directory="$(pwd)/build/server/${revision}/${manual}_manual/"
+    manual_infix="$(tr '[:lower:]' '[:upper:]' <<< ${manual:0:1})${manual:1}"
+    book_file="books/ownCloud_${manual_infix}_Manual.adoc" 
+    nav_file="modules/${manual}_manual/nav.adoc"
+    input_file="books/catfile.$$.adoc"
```

Comment:
Why `catfile` and not `ownCloud_${manual_infix}_Manual`?

Reply:
Good question.